src/solvers/dgmulti/flux_differencing.jl

```diff
@@ -202,11 +191,15 @@ function entropy_projection!(cache, u, mesh::VertexMappedMesh, equations, dg::DG
 
   # TODO: simplices. Address hard-coding of `entropy2cons!`
   apply_to_each_field(mul_by!(Vq), u_values, u)
-  entropy_var_values .= cons2entropy.(u_values, equations)
+  @threaded for i in eachindex(u_values)
```

Comment:
Odd! `eachindex` causes the simulation to crash, but it runs with `Base.OneTo(length(...))`.

Reply:
Which types do `entropy_var_values` and `u_values` have?

Reply:
@ranocha adding a comment on this. Seems like an issue with Polyester.

Reply:
They're both StructArray types I think, so `eachindex` returns iterators over `CartesianIndex`

Reply:
Opened an issue https://github.com/JuliaSIMD/Polyester.jl/issues/37

Mind if I just add a comment linking to the issue for now?